crates/core_arch/src/aarch64/v8.rs

```diff
@@ -11,20 +11,23 @@ use stdarch_test::assert_instr;
 /// Reverse the order of the bytes.
 #[inline]
 #[cfg_attr(test, assert_instr(rev))]
+#[stable(feature = "neon_intrinsics", since = "1.59.0")]
 pub unsafe fn _rev_u64(x: u64) -> u64 {
```

Comment:
Same here, there intrinsics are not covered by the FCP.

Reply:
OK. Unmarked it.